segmentation/include/pcl/segmentation/impl/min_cut_segmentation.hpp

```diff
@@ -76,13 +76,13 @@ pcl::MinCutSegmentation<PointT>::MinCutSegmentation () :
 template <typename PointT>
 pcl::MinCutSegmentation<PointT>::~MinCutSegmentation ()
 {
-  if (search_ != 0)
+  if (search_ != nullptr)
```

Comment:
Here also I see no reason to do any of these `reset`/`clear` operations explicitly.